common/constants/dfget_super_code.go

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package constants
+
+// This file defines the code required for both dfget and supernode.
+
+const (
+	// HTTPError represents that there is an error between client and server.
+	HTTPError = -100
+	// Success represents the request is success.
+	Success = 200
+)
+
+/* the response code from supernode */
+const (
+	CodePeerFinish = 600 + iota
```

Comment:
It's better to assign value to them explicitly. It's easy for us to search error code and avoid automatically changing value when we insert a new code into them.